lock/lock_test.go

```diff
@@ -0,0 +1,46 @@
+package lock
+
+import (
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/stretchr/testify/require"
+	"testing"
+	"time"
+
+	"github.com/gruntwork-io/go-commons/logging"
+	"github.com/stretchr/testify/assert"
+)
+
+var options = Options {
```

Comment:
You want to do this within the TestXXX method below. That way, if there are multiple test methods, each one creates its own `Options`, with different (unique) lock tables / strings.

Reply:
Also, when instantiating structs, its best to pass the field names explicitly, rather than relying on ordering, which is error prone, and hard to read. E.g.,

```go
options := Options {
  Logger: logging.getLogger("test"),
  LockTable: xxx,
  LockString: yyy,
  // etc
}
```

Reply:
that's all very good points - thanks a lot Jim!